### PR TITLE
ZCS:13360 - Add performance script coverage for free tier chat

### PR DESCRIPTION
**Pull request by nikdeshmukh**
Added Performance test coverage for below scenarios:
1. Authentication.
2. Initial loadings
        ▪  Load profile data.
        ▪  Load profile permissions.
        ▪  Load contacts.
        ▪  Load conversations.
3. Send text messages on 1x1 chats.
4. Send file(image & doc) messages on 1x1 chats up to 5MB.
5. Websocket concurrent connections

### CODE REVIEW

File: build.xml [hunk 1]
```diff
@@ -161,351 +164,383 @@
...
-  </target>
-  <target name="zm-sync-common-test">
-    <available file="src/build/jar/zm-sync-common.jar"
-              property="zm-sync-common-jar.present"/>
-  </target>
-  <target name="zm-sync-common" depends="zm-sync-common-test"
-          unless="zm-sync-common-jar.present">
-    <fail message="generic-eas test can not be run unless zm-sync-common.jar is copied to src/build/jar manually"/>
-  </target>
+		    resultlog="${logs}/generic-pop-requests.log">
```
Comment (prashantajari) [inline build.xml:167]: please check indentation

File: tests/generic/eas/profiles/basic.prop [hunk 2]
```diff
@@ -1,9 +1,29 @@
 PROFILE.EAS.type=sequence
 PROFILE.EAS.sequence.1=FolderSync
-PROFILE.EAS.sequence.2=FolderCreate
-PROFILE.EAS.sequence.3=FolderDelete
-PROFILE.EAS.sequence.4=FolderSync
-PROFILE.EAS.sequence.5=SendMail
-PROFILE.EAS.sequence.6=InitialSync
-PROFILE.EAS.sequence.7=DeltaSync
-PROFILE.EAS.sequence.8=Search
+PROFILE.EAS.sequence.2=InitialSync
```
Comment (prashantajari) [inline tests/generic/eas/profiles/basic.prop:3]: Add this as new prop file

File: tests/generic/Chat/Free_Tier_Chat_Solution.jmx [hunk 3]
```diff
@@ -0,0 +1,1682 @@
...
+            <stringProp name="Header.value">Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.75 Safari/537.36</stringProp>
+          </elementProp>
+        </collectionProp>
+      </HeaderManager>
+      <hashTree/>
+      <Arguments guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments">
+          <elementProp name="CHAT_URL" elementType="Argument">
+            <stringProp name="Argument.name">CHAT_URL</stringProp>
+            <stringProp name="Argument.value">test-api.qa.immail.ca</stringProp>
```
Comment (prashantajari) [inline tests/generic/Chat/Free_Tier_Chat_Solution.jmx:28]: get this value from config file

File: build.xml (hunk 1 above)
Comment (nikdeshmukh) [reply build.xml]: indentation corrected:
https://github.com/Zimbra/zm-load-testing/commit/3fa6b3f63e640e103038ac889cb9053730ebbe05#diff-766797f233c18114f9499750cf1ffbf3829aeea50283850619c01bd173132021

File: tests/generic/eas/profiles/basic.prop (hunk 2 above)
Comment (nikdeshmukh) [reply tests/generic/eas/profiles/basic.prop:3]: Done.
https://github.com/Zimbra/zm-load-testing/commit/3fa6b3f63e640e103038ac889cb9053730ebbe05#diff-09c23db38dc4bf525773cd16b2d3a134266e054616e56c3411fa3581b03b5d23

File: tests/generic/Chat/Free_Tier_Chat_Solution.jmx (hunk 3 above)
Comment (nikdeshmukh) [reply tests/generic/Chat/Free_Tier_Chat_Solution.jmx]: Done
https://github.com/Zimbra/zm-load-testing/pull/97/commits/3fa6b3f63e640e103038ac889cb9053730ebbe05#diff-7bafb0399a84b679a993a81161cd8471167e90e1421206522714da88c30cbbe3

**Pull request merged**